docs/self-managed/zeebe-deployment/operations/rebalancing.md

```diff
@@ -0,0 +1,47 @@
+---
+id: rebalancing
+title: "Rebalancing"
+---
+
+Rebalancing is re-electing partition leaders such that they are evenly distributed across all brokers.
+An even leader distribution is beneficial as all brokers share the work of being partition leaders.
+Zeebe will, by default, prefer an even leader distribution when electing new leaders, but will not trigger a re-election unless a leader becomes unavailable.
+When a Zeebe cluster uses an uneven leader distribution, for example caused by losing a leader and thus electing a suboptimal broker as new leader, manually requesting rebalancing can restore the cluster to an even leader distribution.
+
+## Manual Rebalancing
+
+The gateway exposes an HTTP API to request rebalancing. You can use it by `POST`ing to the `/actuator/rebalance` endpoint on the monitoring port of the gateway: 
+
+```bash
+curl -X POST https://{zeebe-gateway}:9600/actuator/rebalance/
+```
+
+The result of this operation is always `200 OK` with no body, even when rebalancing is [not supported](#limitations) by the current configuration or when not all leaders have been contacted.
+
+You can track the rebalancing progress via [metrics](./metrics.md). During the rebalancing, partitions might become unhealthy and can't make progress until a new leader is elected.
+
+### Limitations
+
+Manual rebalancing is done on a best-effort basis.
+Due to the nature of distributed systems, Zeebe can never guarantee a particular distribution and rebalancing cannot circumvent that. 
+
+There are two configurations under which manual rebalancing is supported. 
+
+
+1. **Priority Election** with **Round-Robin Distribution** <br/>
+Priority Election and Round-Robin Distribution are enabled by default. 
```

Comment:
Specify the configuration flags for priority election and distribution here to avoid confusion.

Reply:
I've added a small section below, specifying where to configure priority election and partition distribution